core/pom.xml

```diff
@@ -256,10 +256,18 @@
       <groupId>com.clearspring.analytics</groupId>
       <artifactId>stream</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
+      <artifactId>dropwizard-metrics</artifactId>
+    </dependency>
     <dependency>
       <groupId>io.dropwizard.metrics</groupId>
       <artifactId>metrics-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-jmx</artifactId>
```

Comment:
Why exactly we need `metrics-jmx` module? Is there any transitive dependencies being used?

Reply:
There is a change in the version of codahale-metrics dependency from `3.1.5` to `4.0.5`. In the new version they have separated JMX part as a new module.